background_jobs/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"time"
+
+	"github.com/bitcoin-sv/arc/background_jobs/jobs"
+	"github.com/bitcoin-sv/arc/dbconn"
+	"github.com/go-co-op/gocron"
+	"github.com/rs/zerolog"
```

Comment:
Note that we use `log/slog` in for structured logs in the rest of the code

Reply:
slog has very clumsy API which does not support even formatted string parameters. But if you insist will roll it back to it.